src/components/FoodList/FoodList.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+
+const FoodList = (props) => {
+  const [quantity, setQuantity] = useState(1);
+
+  const addSelectedFood = (food) => {
+    //console.log(food);
+    food.quantity += parseInt(quantity);
```

Comment:
Perhaps change this to the following: 

```
    if (food.quantity) {
      food.quantity += quantity;
    } else {
      food.quantity = quantity;
    }
```

If a new food item is added, this ensures the quantity will be passed to the calories list. Without it, I encounter a "NaN" in the calories.